packages/ui/src/theme/css/component/fieldGroup.scss

```diff
@@ -58,7 +58,7 @@
     @extend %remove-end-radii;
 
     &:not(:focus) {
-      border-inline-end: none;
+      border-inline-end-color: transparent;
```

Comment:
Does this fix https://github.com/aws-amplify/amplify-ui/issues/1634?

Reply:
Oh yeah! It also fixes #1634. Thanks for bringing this up!

Reply:
Want to add that to the description above so it's autoclosed?

Reply:
Yes will do